Stripe/STPShippingAddressViewController.m

```diff
@@ -89,16 +89,8 @@ - (instancetype)initWithConfiguration:(STPPaymentConfiguration *)configuration
         if (shippingAddress != nil) {
             _addressViewModel.address = shippingAddress;
         }
-        else if (prefilledInformation != nil) {
-            STPAddress *prefilledAddress = [STPAddress new];
-            prefilledAddress.country = _addressViewModel.address.country;
-            if (self.configuration.requiredShippingAddressFields & PKAddressFieldEmail) {
-                prefilledAddress.email = prefilledInformation.email;
-            }
-            if (self.configuration.requiredShippingAddressFields & PKAddressFieldPhone) {
-                prefilledAddress.phone = prefilledInformation.phone;
-            }
-            _addressViewModel.address = prefilledAddress;
+        else if (prefilledInformation.shippingAddress != nil) {
```

Comment:
Yeah, this part is pretty redundant. We need to pass in `prefilledInformation` to get `billingAddress` though, so we still need to handle `prefilledInformation.shippingAddress` appropriately.